lib/internal/webstreams/writablestream.js

```diff
@@ -263,8 +263,6 @@ class WritableStream {
     this[kState].transfer.readable = readable;
     this[kState].transfer.promise = promise;
 
-    setPromiseHandled(this[kState].transfer.promise);
```

Comment:
This is unnecessary because it is called in `newCrossRealmReadableStream`.
https://github.com/nodejs/node/blob/880c90c873c3691485619c8167b73ccb2f843076/lib/internal/webstreams/transfer.js#L264-L272